kalamine/tpl/base.ahk

```diff
@@ -61,10 +61,22 @@ SetTimer, ShowTrayTip, -1000  ; not working
 
 global DeadKey := ""
 
+UnicodeUpperIfCapsLock(unicode:="") {
+  CapsLockOn := GetKeyState("CapsLock", "T") 
+  if %CapsLockOn% {
+    unicode := Chr("0x" SubStr(unicode, 3, 4))
```

Comment:
This would break if the character is sent as a single byte — which is something kalamine allows to do to ease the debug, see the `esc_all` constant in `template.py` line 120.